### DISCUSSION TITLE
Direct link to a specific feature

### DISCUSSION THREAD

**Discussion by abjanssen** [GMOD/jbrowse-components#4365] [General]
I am looking for a way to make a link (that could be incorporated in an external website), that when clicked, would take you to the right genome and place in the genome where a specific feature is located in JBrowse 2.

For example: 
In PneumoWiki, a direct link to the current version of PneumoBrowse is incorporated (look up "PneumoBrowse" on this page: https://pneumowiki.med.uni-greifswald.de/SPV_1113). That link will take you to the place in the D39V genome where this feature (SPV_1113) is located.
However, that version of PneumoBrowse is made in JBrowse 1, and we are now in the process of switching to JBrowse 2, and thus that link will break. 

I could not find any method in the documentation to establish something similar in JBrowse 2. I've tried the bookmarks, but this seems to be different in functionality. A shared URL might contain all the data necessary, but the pattern encoded in the long version of this shared URL does not seem to be easily adaptable to a range of features.
Is there any way we could keep this link alive, but have it point to a JBrowse 2 session, instead of JBrowse 1?

Thank you kindly for any help!

**Comment by cmdcolin** [comment]
there are simplified url query parameters that are designed to help with this https://jbrowse.org/jb2/docs/urlparams/

e.g. http://host/jbrowse2/?config=test_data/config.json&loc=chr1:6000-7000&assembly=hg19&tracks=gene_track,vcf_track

**Reply by cmdcolin** [reply]
note that gene names are also allowed in the &loc= if there is a gene name search index (e.g. jbrowse text-index was run), but supplying the exact location will avoid the need for jbrowse to perform the gene name lookup and might be more reliable

**Comment by abjanssen** [comment]
Thank you very much, this is very helpful!
